docs/en/get-started/using-platformio.rst

```diff
@@ -0,0 +1,57 @@
+.. _platformio:
+
+.. image:: image:: ../../_static/platformio-logo.png
+  :target: https://platformio.org/?utm_source=docs.espressif.com
+
+.. contents::
+    :local:
+
+What is PlatformIO?
+~~~~~~~~~~~~~~~~~~~
+
+`PlatformIO <https://platformio.org/?utm_source=docs.espressif.com>`__
+is a cross-platform embedded development environment with ESP-IDF support
+maintained by its developers.
+
+Since ESP-IDF support within PlatformIO is not maintained by the ESP-IDF team,
+please report any issues with PlatformIO directly to its developers in
+`the official PlatformIO repositories <https://github.com/platformio>`_.
+
+A detailed overview of the PlatformIO ecosystem and its philosophy can be
+found in `the official PlatformIO documentation <https://docs.platformio.org/en/latest/what-is-platformio.html?utm_source=docs.espressif.com>`_.
+
+Installation
+~~~~~~~~~~~~
+
+* `PlatformIO IDE <https://platformio.org/platformio-ide?utm_source=docs.espressif.com>`_
+  is a toolset for embedded C/C++ development available on Windows, macOS and Linux
+  platforms
+* `PlatformIO Core (CLI) <https://docs.platformio.org/en/latest/core/index.html?utm_source=docs.espressif.com>`_
+  is a command-line tool that consists of multi-platform build system, platform and
+  library managers and other integration components. It can be used with a variety of
+  code development environments and allows integration with cloud platforms and web
+  services
+
+Configuration
+~~~~~~~~~~~~~
+
+Please go through `the official PlatformIO configuration guide for ESP-IDF framework <https://docs.platformio.org/en/latest/frameworks/espidf.html?utm_source=docs.espressif.com#configuration>`_.
```

Comment:
suggest dropping "framework", otherwise this reads "iot development framework framework"

Reply:
By the way, one question here is how to link to the correct version of ESP-IDF support in PlatformIO. It seems that this link will always bring the reader to the "latest" version, even if they were reading the docs for, say, IDF v3.3.2.

Reply:
> suggest dropping "framework", otherwise this reads "iot development framework framework"

Fixed

> By the way, one question here is how to link to the correct version of ESP-IDF support in PlatformIO. It seems that this link will always bring the reader to the "latest" version, even if they were reading the docs for, say, IDF v3.3.2.

I've added to PlatformIO docs an additional section for ESP-IDF v3.x, so when readers follow the links from this page they will see what versions of ESP-IDF can be used with PlatformIO.